src/Microsoft.DotNet.Tools.Pack/Properties/launchSettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "profiles": {
+    "dotnet-pack": {
+      "executablePath": "..\\..\\artifacts\\win7-x64\\stage2\\bin\\dotnet-pack.exe"
```

Comment:
yea, I'm concerned about portability. For the time being, I'm ok with it. we'll learn something in the process.